skynet.yaml

```
@@ -0,0 +1,15 @@
+name: validate_js_bundle
+description: Verifies the JS bundle exists
```

I think this check could be confusing for future contributors. Could we clarify what's being tested here, and add some context on why it was added?
```suggestion
description: Verifies the JS bundle is checked in - this check is mainly in place so we have at least one Skynet config
```